app/src/main/java/com/hms/explorehms/pushkit/MainActivityPushKit.java

```diff
@@ -297,12 +300,15 @@ private void setAAID(boolean isGet) {
             new Thread(() -> {
```

Comment:
starting thread in scope is so dangerous,  you have to assign a parameter and stop onDestroy method, Because when page destroy, thread might still work

Reply:
I fixed this problem